test_suite/cloud/test_azure.py

```diff
@@ -313,8 +313,17 @@ def test_rhui_certificate_date(self, host):
             cert_found = False
 
             if test_lib.is_rhel_sap(host):
-                cert_file = '/etc/pki/rhui/product/content-sap-ha.crt'
-                cert_found = host.file(cert_file).exists
+                possible_sap_cert_files = [
+                        '/etc/pki/rhui/product/content-sap-ha.crt',
+                        '/etc/pki/rhui/product/content-base-sap-ha.crt'
+                        ]
+                for cert in possible_sap_cert_files:
+                    if host.file(cert).exists:
```

Comment:
Let's put this outside the `else` statement, so that it can always apply based on the cert files that need to be checked.